docusaurus/docs/guides/deployments/30-kubernetes/kubernetes-console.mdx

```diff
@@ -1,9 +1,48 @@
 ---
 sidebar_position: 30
 sidebar_label: Console
-title: Install the Console in Kubernetes
+title: Kubernetes Console
 ---
 
-import ConsoleHelmChartReadme from '/_remotes/helm-charts/charts/ziti-console/README.md';
+Use the console by navigating to the controller's address with path `/zac/` in a web browser. If you published the controller at `ziti.example.com:443`, then the console URL is `https://ziti.example.com/zac/`.
```

Comment:
there are no instructions on this page whatsoever, it's much more 'referernce' than guide and doesn't seem to fit here?

Reply:
We could make this part of the controller guide or redirect this location where the k8s guide for the standalone node server was. I'm happy with the current organization for now because it's relevant to deployments and provides the information a deployer needs to utilize the console.